src/main/java/frc/robot/Robot.java

```diff
@@ -24,13 +26,22 @@ public void robotPeriodic() {
   }
 
   @Override
-  public void disabledInit() {}
+  public void disabledInit() {
+    m_neutralModeTimer.start();
+  }
 
   @Override
-  public void disabledPeriodic() {}
+  public void disabledPeriodic() {
+    // if 10 seconds have passed since disabling, set the neutral mode to coast
+    if (m_neutralModeTimer.get() > 10) {
```

Comment:
You also you `hasElapsed` if you prefer, also 10 should be a constant (in Constants.java)